frontend/javascripts/oxalis/model/accessors/dataset_layer_transformation_accessor.ts

```diff
@@ -364,15 +381,15 @@ function isTranslationOnly(transformation?: AffineTransformation) {
   return scale.equals(NON_SCALED_VECTOR) && quaternion.equals(IDENTITY_QUATERNION);
 }
 
-function isRotationOnly(transformation?: AffineTransformation) {
+function isRotationAndMirrorMaybeOnly(transformation?: AffineTransformation) {
```

Comment:
how about: `isOnlyRotatedOrMirrored`? I think "or" makes more sense and the "maybe" part can be omitted?